vars/checkChangeInDevelopment.groovy

```diff
@@ -0,0 +1,89 @@
+import com.sap.piper.GitUtils
+import groovy.transform.Field
+
+import com.sap.piper.ConfigurationMerger
+import com.sap.piper.cm.ChangeManagement
+import com.sap.piper.cm.ChangeManagementException
+
+@Field def STEP_NAME = 'checkChangeInDevelopment'
+
+@Field Set parameterKeys = [
+    'changeDocumentId',
+    'cmClientOpts',
+    'credentialsId',
+    'endpoint',
+    'failIfStatusIsNotInDevelopment',
+    'git_from',
+    'git_to',
+    'git_label',
+    'git_format'
+  ]
+
+@Field Set stepConfigurationKeys = [
+    'changeDocumentId',
+    'cmClientOpts',
+    'credentialsId',
```

Comment:
Could you change it to cmCredentials like the other steps?

Reply:
Yes, this I understand it, and I wrote it like this in the beginning, but since you were preparing a new feature for the configuration, in order to configure global properties, I thought then the context would be missing. Will not we make this properties global?

Reply:
Also, if you don't change it, you have to change the documentation.

Reply:
No, no global variables. Providing a - lets say - landscape configuration layer we should fit into the configuration framework. My suggestion is to extend the already existing ```general```section. Eg like ```general/<system>``` where general could be ```cm``` or ```git``` for denoting a companies git server  or ```nexus``` for describing a binary repository receiving and providing artifacts. But so far there is no decission about how to populate/enrich the configuration framework. Currently it is only about providing the missing layer.

Reply:
Thanks for the hint, docu needs to be updated also, yes.